aws_lambda_powertools/utilities/data_classes/kinesis_firehose_event.py

```diff
@@ -77,6 +196,26 @@ def data_as_json(self) -> dict:
             self._json_data = self._json_deserializer(self.data_as_text)
         return self._json_data
 
+    def create_firehose_response_record(
+        self,
+        result: Literal["Ok", "Dropped", "ProcessingFailed"],
```

Comment:
```suggestion
        result: Literal["Ok", "Dropped", "ProcessingFailed"] = "Ok",
```

You can confirm with the customer, but I suspect this is 70+% of the use cases. If they want to drop, it'll be clearer when reading a code that something it's not the standard behaviour.

Similarly, ProcessingFailed will likely appear during a try/catch only. We could also optimize here with another method but without customer feedback is an overkill atm ;)

Reply:
This is a really good optimization. Thanks!